### DISCUSSION TITLE
not able to Save and Continue

### DISCUSSION THREAD

**Discussion by SathyaSathish16** [devchallenges-io/curriculum#5] [General]
This is my first challenge. When submiting solutions , i'm not able to save and continue to second tab from solution details tab. why?

![Screenshot 2024-10-08 131500](https://github.com/user-attachments/assets/4e01bc56-b9bb-48d1-a5a7-adea340a6033)

**Comment by nghiemthu** [comment]
Hi @SathyaSathish16, you need to provide description longer than 50 characters. I see that there is some UX issue here, I'll work on this to improve the experience 👍
